script/utils/check-task-numbering.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
```

Comment:
Best practice to ensure safe scripts
```suggestion
#!/bin/bash
set -euo pipefail

```